src/Pos/Wallet/Web/Api.hs

```diff
@@ -108,103 +108,103 @@ type TestReset =
     :> WRes Post ()
 
 -------------------------------------------------------------------------
--- Wallet sets
+-- Wallets
 -------------------------------------------------------------------------
 
-type GetWalletSet =
+type GetWallet =
        "wallets"
     :> "sets"
     :> Capture "walletSetId" (CId Wal)
     :> WRes Get CWallet
 
-type GetWalletSets =
+type GetWallets =
        "wallets"
     :> "sets"
     :> WRes Get [CWallet]
 
-type NewWalletSet =
+type NewWallet =
        "wallets"
     :> "sets"
     :> "new"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] CWalletInit
     :> WRes Post CWallet
 
-type RestoreWalletSet =
+type RestoreWallet =
        "wallets"
     :> "sets"
     :> "restore"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] CWalletInit
     :> WRes Post CWallet
 
-type RenameWalletSet =
+type RenameWallet =
        "wallets"
     :> "sets"
     :> "rename"
-    :> Capture "walletSetId" (CId Wal)
+    :> Capture "walletId" (CId Wal)
     :> Capture "name" Text
     :> WRes Post CWallet
 
-type DeleteWalletSet =
+type DeleteWallet =
        "wallets"
     :> "sets"
     :> Capture "walletSetId" (CId Wal)
     :> WRes Delete ()
 
-type ImportWalletSet =
+type ImportWallet =
        "wallets"
     :> "sets"
     :> "keys"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] Text
     :> WRes Post CWallet
 
-type ChangeWalletSetPassphrase =
+type ChangeWalletPassphrase =
        "wallets"
     :> "sets"
```

Comment:
we should also change the rest api names. "sets" shouldn't exist. We should reflect this change in both Api.hs and BackendApi.purs